extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -352,6 +352,9 @@ private function getMainPageData() {
 		try {
```

Comment:
we should not reuse try's for different calls
as this will make stuff harder to debug
please add another try block if needed